cfme/tests/containers/test_reload_button_provider.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from utils.appliance.implementations.ui import navigate_to
+from cfme.containers.provider import ContainersProvider
+from utils import testgen, version
+from utils.version import current_version
```

Comment:
You import version, and version.current_version separately, and then call current_version both using the imported module and the function. Pick one way or the other, but since its the only function from version that you're using, I'd recommend keeping the function import and dropping the version import from line 5.